docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/test-coverage-is-common-e-com-flow-covered.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Test coverage: Is common E-com flow covered?
+description: This document allows you to assess if Common E-com flow is covered with tests.
+template: howto-guide-template
+---
+
+# Test coverage: Is common E-com flow covered?
+
+{% info_block infoBox %}
+
+Resources: Backend
+
+{% endinfo_block %}
+
+## Description
+
+Do a code review of the archived project received as part of the prerequisites and identify if the following
+functionalities were covered with tests:
+* PLP (product listing page)
+* PDP (product detail page)
+* Add to Cart
+* Checkout (test steps)
```

Comment:
What are the test steps?

Reply:
address -> shipment -> payment -> place order